test/Prompt/DontWrap.C

```diff
@@ -172,7 +172,7 @@ const Test2::Nested::Value::type & Test2::Nested::reference() const {
 Test2::Nested Nest2(80);
 // CHECK: Nested::Nested(160)
 Nest2.simpleAdd(3)
-// CHECK: (Test2::Nested::Value::type) 83
+// CHECK: (int) 83
```

Comment:
Well this is a problem on Windows.
[cling] std::string s("T")[0]
(ValueWrapper<char>::value_type&) 'T'

'ValueWrapper' may not be the name but it really obfuscates a lot of types that way.
Is there a reason to not use the canonical type?
To me it's a lot more legible when printing the canon type, and can easily imagine templated structs that grow the line phenomenally.